boundary_layer_default_plugin/config/operators/gcp_pubsub_pull_sensor.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2020 Etsy Inc.
```

Comment:
Should we move this out of this PR, as it is included in https://github.com/etsy/boundary-layer/pull/87?

Reply:
Yes, please.  thanks!

Reply:
I reset to before that addition and reapplied my changes.